Documentation/Groups/Groups.md

```diff
@@ -20,14 +20,15 @@ Groups in workbooks are useful for several things:
 3. performance - in cases where you have a very large templates with many sections or tabs, you can convert each section into it's own sub-template and use groups to load all the sub-templates within the top level template. In these cases, the contents of the sub-templates won't load or run until a user makes those groups visible. See [How to split a large template into many templates](#how-to-split-a-large-template-into-many-templates) for more information.
 
 ## Scoping
-Note: at the current time, a group is treated as a new "scope" in the workbook. Any parameters created in the group are only visible *inside the group*. This is also true for things like merge, they can only see data inside their group or at the parent level.
+By default a group is treated as a new "scope" in the workbook. Any parameters created in the group are only visible *inside the group*. This is also true for things like merge, they can only see data inside their group or at the parent level.
+You can change this behavior for parameters by going into the `Advanced settings` for the group, and enabling `Export parameters inside this group to be visible outside the group` option. When this option is enabled, parameters inside a group will be visible below that group, like other standard parameters.
```

Comment:
should we mention anything about scoping conflicts?

Reply:
@vutran01 what specifically do you mean by "scoping conflicts"